lib/elixir/test/elixir/protocol/consolidation_test.exs

```diff
@@ -129,12 +129,12 @@ defmodule Protocol.ConsolidationTest do
     assert {{:function, :ok, 1}, _, ["ok(term)"], %{"en" => "Ok"}, _} = ok_doc
   end
 
-  test "consolidation keeps deprecated" do
+  test "consolidation keeps chunks" do
     deprecated = [{{:ok, 1}, "Reason"}]
     assert deprecated == Sample.__info__(:deprecated)
 
-    {:ok, {Sample, [{'ExDp', deprecated_bin}]}} = :beam_lib.chunks(@sample_binary, ['ExDp'])
-    assert {:elixir_deprecated_v1, deprecated} == :erlang.binary_to_term(deprecated_bin)
+    {:ok, {Sample, [{'Docs', docs_bin}]}} = :beam_lib.chunks(@sample_binary, ['Docs'])
```

Comment:
I think these two lines could be removed, they are tested in the previous test. Maybe something like?

```elixir
test "consolidation keeps deprecated info" do
   assert {{:ok, 1}, "Reason"}] == Sample.__info__(:deprecated)
end
```

Reply:
Thanks, I rewrote it slightly to make more sense.